source/deploying-with-kubernetes/kubernetes-clean-up.rst

```diff
@@ -0,0 +1,261 @@
+
+4. Finally delete the persistent volumes.
+
+    .. code-block:: bash
+
+        ubuntu@k8s-control-server:~$ kubectl get persistentvolume
+        NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS        CLAIM                                                         STORAGECLASS             REASON    AGE
+        pvc-024466da-f7c5-11e8-b9b8-022ada63b4ac   10Gi       RWO            Retain           Released      wazuh/wazuh-manager-worker-wazuh-manager-worker-1-0           gp2-encrypted-retained             6d
+        pvc-b3226ad3-f7c4-11e8-b9b8-022ada63b4ac   30Gi       RWO            Retain           Bound         wazuh/wazuh-elasticsearch-wazuh-elasticsearch-0               gp2-encrypted-retained             6d
+        pvc-fb821971-f7c4-11e8-b9b8-022ada63b4ac   10Gi       RWO            Retain           Released      wazuh/wazuh-manager-master-wazuh-manager-master-0             gp2-encrypted-retained             6d
+        pvc-ffe7bf66-f7c4-11e8-b9b8-022ada63b4ac   10Gi       RWO            Retain           Released      wazuh/wazuh-manager-worker-wazuh-manager-worker-0-0           gp2-encrypted-retained             6d
+
+        ubuntu@k8s-control-server:~$ kubectl delete persistentvolume pvc-b3226ad3-f7c4-11e8-b9b8-022ada63b4ac
+
+.. warning::
+    Do not forget to delete the volumes manually in AWS.
+
+Logstash
+--------
+
+To clean the Logstash installation remove the Logstash deployments and services.
+
+1. The first step is to remove the services related to Logstash.
+
+    .. code-block:: bash
+            
+        ubuntu@k8s-control-server:~$ kubectl get services --namespace wazuh
+        NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP        PORT(S)                          AGE
+        kibana                ClusterIP      172.20.121.19    <none>             5601/TCP                         6d
+        logstash              ClusterIP      172.20.160.68    <none>             5000/TCP                         6d
+        wazuh-nginx           LoadBalancer   172.20.166.239   internal-ac0c...   80:30409/TCP,443:32575/TCP       6d
+
+        ubuntu@k8s-control-server:~$ kubectl delete service logstash --namespace wazuh
+
+2. And the second and last step is to eliminate the deployment.
+
+    .. code-block:: bash
+
+        ubuntu@k8s-control-server:~$ kubectl get deploy --namespace wazuh
+        NAME             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+        wazuh-kibana     1         1         1            1           6d
+        wazuh-logstash   1         1         1            1           6d
+        wazuh-nginx      1         1         1            1           6d
+
+        ubuntu@k8s-control-server:~$ kubectl delete deploy wazuh-logstash --namespace wazuh
+
+Kibana and Nginx
+----------------
+
+To clean the Kibana and Nginx installation remove their deployments and services.
+
+1. The first step is to remove the pods corresponding to Kibana and Nginx.
+
```

Comment:
It is not necessary to delete the pods, when the sts are deleted they will be deleted. I miss to put it in the previous revision.